govdelivery/api.py

```diff
@@ -130,3 +130,29 @@ def set_subscriber_answers_to_question(self, contact_details, question_id, answe
 
         payload = xml_payloads.free_response_to_question(question_id_encoded, answer_text)
         return self.call_api(path, "put", payload)
+
+    set_subscriber_answer_to_free_response_question = set_subscriber_answers_to_question
```

Comment:
I see why you want to make this change; I'm not sure I would do it this way without maybe also including [some kind of deprecation warning](https://stackoverflow.com/questions/2536307/decorators-in-the-python-standard-lib-deprecated-specifically/30253848#30253848). There aren't many other uses of this library on public GitHub, but [there are a few](https://github.com/search?q=set_subscriber_answers_to_question&type=Code).

I might revert this particular change and handle it as a separate PR, especially given the lack of tests on this repo. If the name of the previous method is confusing, we could also add a docstring.

Reply:
Removed for a future PR.